app/graphql/types/query_type.rb

```diff
@@ -199,6 +199,14 @@ def static_content_with_data_type(data_type, name, version = nil)
         static_contents = StaticContent.where(data_type: data_type, name: name)
         static_content = find_static_content(static_contents, name, version)
 
+        # fallback for old app versions, that does not send versions, thus handle responses as
+        # string and parses on mobile
+        if version.nil?
+          return static_content if static_content.present?
+
+          return { content: data_type == "html" ? "" : {}, name: "not found" }
+        end
+
```

Comment:
do we want this in master forever or just use it temporarily in several (old) release branches?

Reply:
I think we can keep it